nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/cmd/exec/internal/DefaultCommandPathVariableResolver.java

```diff
@@ -179,4 +196,37 @@ protected String mapColElem(Param<?> commandParam, String pathToResolve) {
 		// throw ex ..or.. blank??
 		return "";
 	}
+	
+	/**
+	 * 
+	 * @param commandParam
+	 * @param pathToResolve
+	 * @return
+	 * @since 1.1.11
+	 */
+	protected Object mapCrossDomain(Param<?> commandParam, String pathToResolve) {
+		String hostUri = commandParam.getRootExecution().getRootCommand().buildUri(Type.AppAlias);
+		StringBuilder uri = new StringBuilder(hostUri);
+		uri.append(pathToResolve);
+		/* action */
+		uri.append(Constants.SEPARATOR_URI.code).append(Action._get.name()); // /_get
+		
+		/* behavior */
+		uri.append(Constants.REQUEST_PARAMETER_MARKER.code).append(Constants.MARKER_URI_BEHAVIOR.code)
```

Comment:
Can CommandBuilder be used for first creating the Command from base URI and then use the chaining methods to add action, behavior, etc. That way we can avoid string based concatenation logic to remain within a centralized CommandBuilder class instead of each client to rebuild the same logic.